src/js/components/search/visualizations/geo/MapLayerToggle.jsx

```diff
@@ -61,7 +61,7 @@ const MapLayerToggle = (props) => {
                         <TooltipWrapper
                             icon="info"
                             className={props.className}
-                            tooltipPosition={(window.innerWidth >= tabletScreen && window.innerWidth <= mLargeScreen) ? 'left' : 'right'}
+                            tooltipPosition={(window.innerWidth >= tabletScreen && window.innerWidth <= xLargeScreen) ? 'left' : 'right'}
```

Comment:
just confirming we still need this to say xlargescreen? i think we do, just confirming

Reply:
yes, that needs to xLargeScreen, so we send 'left' all the way up to 1640px; if we send 'right' btw 1400-1640 the caret is in the wrong place